src/components/checkbox/checkbox.scss

```diff
@@ -14,10 +14,21 @@
     flex-wrap: wrap;
 
     .mdc-checkbox {
+        .mdc-checkbox__native-control {
+            &[disabled],
+            &:enabled {
+                &:not(:checked):not(:indeterminate) {
+                    ~ .mdc-checkbox__background {
+                        background-color: rgb(var(--contrast-300));
+                    }
+                }
+            }
+        }
+
         &.mdc-checkbox--invalid {
             .mdc-checkbox__native-control:enabled:not(:checked):not(:indeterminate)
                 ~ .mdc-checkbox__background {
-                border-color: var(--lime-error-text-color);
+                border-color: var(--lime-red-text-color);
```

Comment:
Also, this change doesn't seem to be related to the background color change? Should it perhaps be in a separate commit?